File: aoc2025/java/src/test/java/com/peeekay/aoc2025/java/Day01Test.java
package com.peeekay.aoc2025.java;

import com.peeekay.aocCommon.AOCPuzzle;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class Day01Test {
    final AOCPuzzle day = new Day01(true);

    @Test
    void part1() {
        Assertions.assertEquals(3L, day.part1());
    }

    @Test
    void part2() {
        Assertions.assertEquals(6, day.part2());
    }
}

File: test/com/peeekay/aoc2024/kotlin/Day01Test.kt
package com.peeekay.aoc2024.kotlin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

@DisplayName("Day 01")
internal class Day01Test {

    @Test
    fun `part1 test data solve gives correct answer`() {
        val day = Day01(isTest = true)
        val answer = day.partOne()

        assertEquals(11, answer)
    }

    @Test
    fun `part2 test data solve gives correct answer`() {
        val day = Day01(isTest = true)
        val answer = day.partTwo()

        assertEquals(31, answer)
    }

}
